src/commands/commerce/extension/map.ts

```diff
@@ -0,0 +1,154 @@
+            throw new SfdxError(errorMsg);
+        }
+        return registeredExternalServiceId;
+    }
+
+    private validateStoreId(storeName: string, storeId: string, userName: string): string {
+        let fResult: Result<QueryResult>;
+        let duplicate: boolean;
+        if (storeId === undefined) {
+            try {
+                fResult = forceDataSoql(`SELECT Id FROM WebStore WHERE Name='${storeName}'`, userName);
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                if (fResult.result.totalSize > 1) {
+                    duplicate = true;
+                }
+            } catch (e) {
+                throw new SfdxError(
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                    msgs.getMessage('extension.map.errStoreName', [storeName, '\n', e.message])
+                );
+            }
+            // check to see if multiple stores with the same name
+            if (duplicate === true) {
+                throw new SfdxError(
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                    msgs.getMessage('extension.map.multiple', [storeName])
+                );
+            } else {
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+                storeId = fResult.result.records[0].Id;
+            }
+        } else {
+            try {
+                storeId = forceDataSoql(`SELECT Id FROM WebStore WHERE Id='${storeId}' LIMIT 1`, userName).result
+                    .records[0].Id;
+            } catch (e) {
+                throw new SfdxError(
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                    msgs.getMessage('extension.map.errStoreId', [storeId, '\n', e.message])
+                );
+            }
+        }
+        return storeId;
+    }
+
+    private getInsertedRecord(storeid: string, registeredExternalServiceId: string): string {
+        const StoreIntegratedTable = forceDataSoql(
+            `SELECT Id,Integration,ServiceProviderType,StoreId from StoreIntegratedService WHERE StoreId= '${storeid}' and Integration='${registeredExternalServiceId}' limit 1`
+        );
+        for (const element of StoreIntegratedTable.result.records) {
+            const finalTable = {
+                Id: element['Id'],
+                Integration: element['Integration'] as string,
+                StoreId: element['StoreId'] as string,
+            };
+            const returnResult = `${JSON.stringify(finalTable, null, 4)}\n`;
+            this.ux.log(returnResult);
+            this.ux.log(
+                msgs.getMessage('extension.map.savingConfigIntoConfig', [
+                    this.flags['registered-extension-name'],
+                    'to your webstore',
```

Comment:
can we add the webstore to the end of the message since they are mapping the extension to a specific webstore